ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel.yaml

```diff
@@ -1,33 +1,33 @@
 base_images:
   base:
-    name: "4.13"
+    name: "4.16"
```

Comment:
/lgtm

nit: can you also change `OO_CHANNEL: stable-1.3` to `OO_CHANNEL: stable` 

We are only using `stable` channel name.